core/src/main/antlr4/com/databricks/labs/remorph/parsers/snowflake/SnowflakeParser.g4

```diff
@@ -1601,16 +1620,22 @@ callerOwner: CALLER | OWNER
 executaAs: EXECUTE AS callerOwner
     ;
 
-procedureDefinition: string | DBL_DOLLAR
+procedureBody: (let | call | executeImmediate | string)*
+    ;
+
+procedureDefinition: DBL_DOLLAR | declareCommand? BEGIN procedureBody returnStatement END SEMI
```

Comment:
```suggestion
procedureDefinition: DBL_DOLLAR | declareCommand? BEGIN procStatement+ END SEMI
```
there might be multiple `returnStatement` in the body, it's not necessary to have it always the last one. can we just embed the `procStatement+`?